src/components/MoneyReportHeader.tsx

```diff
@@ -1457,9 +1462,21 @@ function MoneyReportHeader({
                     onSelected: showDeleteModal,
                 };
             }
+            if (option.text === translate('search.bulkActions.reject')) {
```

Comment:
NAB since I see that we are doing this above for delete as well, but determining the action for a button based on the text of the button seems like a strange pattern

Reply:
I've pushed code to handle by value